src/Pages/wonkeunchoi/Main/Components/Section/Article/Comment.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+class Comment extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  render() {
```

Comment:
```suggestion
class Comment extends React.Component {
  constructor(props) {
  render() {
```
사용하지 않는 constructor는 제거해주세요!